test/commands/add/activity_spec.rb

```diff
@@ -51,5 +51,267 @@
     end
   end
 
+  describe "adding default location" do
+    describe "when it is the latest activity" do
+      subject { run_cmd("add activity Moved to _Paris_") }
+      let(:content) do
+        <<-FILE
+### Activities:
+- #{preceding_activity}
+
+### Notes:
+
+### Friends:
+
+### Locations:
+FILE
+      end
+
+      describe "when a default location has never been set before" do
+        let(:preceding_activity) { "2016-01-01: Had dinner in _Berlin_." }
+
+
+        it 'prints "Default location set to" output message' do
+          assert_default_location_output('Default location set to: "Paris"')
+        end
+      end
+
+      describe "when preceding default location is different" do
+        let(:preceding_activity) { "2016-01-01: Moved to _Berlin_." }
+        it 'prints "Default location set to" output message' do
+          assert_default_location_output('Default location set to: "Paris"')
+        end
+      end
+
+      describe "when preceding default location is the same" do
+        let(:preceding_activity) { "2016-01-01: Flew to _Paris_." }
+
+        it 'prints "Default location already set to" output message' do
+          assert_default_location_output('Default location already set to: "Paris"')
+        end
+      end
+    end
+
+    describe "when it is not the latest activity" do
+      subject { run_cmd("add activity 2009-01-01: Moved to _Paris_") }
+      let(:content) do
+  <<-FILE
+### Activities:
+- #{proceeding_activity}
```

Comment:
It took me a second to see that this was different from `preceeding_activity`. How would you feel about changing the name to something like `following_activity`?

Reply:
Ha yes I got confused sometimes too! I was rushing a bit with variable names 😅 . `following_activity` is perefct 👍